website/templates/deviceactions.html

```diff
@@ -0,0 +1,123 @@
+{% extends "base.html" %}
+{% block page_content %}
+    <h1> Motion Sensor Action Settings </h1>
+    <br></br>
+    <hr></hr>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet"/>
+    <div class="container">
+
+        <form class="form-horizontal" role="form">
+
+            <div class="form-group">
+                <label class="col-sm-4 control-label"> Motion detection (on/off): </label>
+                <div class="col-sm-8">
+                    <label class="radio-inline"> <input type="radio" name="MotionSensor" id="MotionSensor" value="ON"
+                                                        checked> Motion Sensor is ON</label>
+                    <label class="radio-inline"> <input type="radio" name="MotionSensor" id="MotionSensor" value="OFF">
+                        Motion Sensor is OFF </label>
+
+                </div>
+            </div>
+
+            <ul>
+            </ul>
+            <hr>
```

Comment:
You don't have to close hr and br tags. If you want to, you can write <br/> and <hr/>